### PR TITLE
Closes #67

### LINKED ISSUE #67: Search engine - "Archived cards" list
Exclude archived cards from search unless the archive list is displayed.

### PR DESCRIPTION
**Pull request by vrialland**
Note: indexes must be updated before it can be tested

### CODE REVIEW

**Comment by rtxm** [comment]
What's happening when the card is archived by clicking on its "delete" action?

**Comment by vrialland** [comment]
It just disappear as wanted, but the number of hits isn't updated.
It happens as well if you add a card that matches the search.
I'll check that.

**Comment by vrialland** [comment]
Cards are now unindexed properly and search is refreshed:
- After showing/hiding archive column
- After adding a new card
- After editing a card
- After purging archive column

File: kansha/card/fts_schema.py [hunk 1]
```diff
@@ -33,4 +34,5 @@ def from_model(cls, card):
                    comments='\n'.join(c.comment for c in card.comments),
                    lists='\n'.join(unicode(cl) for cl in card.checklists),
                    tags=' '.join(t.title for t in card.labels),
-                   board_id=card.column.board.id)
+                   board_id=card.column.board.id,
+                   archive=card.column.archive)
```
Comment (rtxm) [inline kansha/card/fts_schema.py]: A card can be archived, but is not an archive. So the propery would be more appropriately named "archived" or "in_archive" for clarity.

**Comment by rtxm** [comment]
There is still an issue when I archive a card by clicking on the "delete" button in edition mode : it is still counted in the number of matches.

**Comment by vrialland** [comment]
Fixed.

**Pull request merged**